docs/managing-policy.md

```diff
@@ -42,3 +42,8 @@ To receive notifications via third-party services, like Slack or email:
 
 When Semgrep CI next runs and finds a result, the configured services will receive the finding.
 
+# Fail open/close
+
+By default, when semgrep-action fails to scan code for some reason (e.g., an invalid config), it will exit with a return code greater than 1. If you do not want this kind of failure to block CI from passing, you can set semgrep-action to fail open. Go to [https://semgrep.dev/manage/projects](https://semgrep.dev/manage/projects) and on the desired repository edit, select, and save the checkbox under the Fail Open column.
```

Comment:
@pabloest Should we standardize on "Semgrep CI" rather than `semgrep-action` here?

@brendongo Is there a reference for what "fail open" means? I believe it's a term of art that may be helpful to explain or link to.

Rather than "fails to scan code", what do you think of language like "internal Semgrep error"? It feels more concise/accurate than than "for some reason" ;)

Reply:
@dlukeomalley I think yes, and also think we should probably do that more consistently? Perhaps even rename the action’s repo? (something for another day though)